bitnami/harbor/templates/registry/registry-cm.yaml

```diff
@@ -125,7 +125,7 @@ data:
         region: {{ .Values.persistence.imageChartStorage.oss.region }}
         bucket: {{ .Values.persistence.imageChartStorage.oss.bucket }}
         {{- if .Values.persistence.imageChartStorage.oss.endpoint }}
-        endpoint: {{ .Values.persistence.imageChartStorage.oss.endpoint }}
+        endpoint: {{ .Values.persistence.imageChartStorage.oss.bucket }}.{{ .Values.persistence.imageChartStorage.oss.endpoint }}
```

Comment:
Not sure about this change, does it work with minio as S3 backend ?

Reply:
For context, this change refers to: https://github.com/goharbor/harbor-helm/pull/675

Regarding your question, minio (or any other `S3` backend) should not be affected by this change as this only applies if the selected storage type for images and charts is `OSS` (and not `S3`) 🙂